android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -323,12 +324,6 @@ private boolean isPlayingAd() {
         return player != null && player.isPlayingAd();
     }
 
-    @Override
-    public void setId(int id) {
```

Comment:
If I remember well this was necessary to support multiple <Video in the same page :/ but maybe I am wrong ...

Reply:
View id will be inject within ReactExoplayerVIewManager.addEventEmitters method from this PR
fyi) https://github.com/TheWidlarzGroup/react-native-video/pull/3962/files#diff-910ac265923b0ad5faf4bbeb2cb1742f2608233f22f952c323925141f1cd8d8fR279-R285